core/object.h

```diff
@@ -760,49 +766,63 @@ void postinitialize_handler(Object *p_object);
 
 class ObjectDB {
 
-	struct ObjectPtrHash {
```

Comment:
Doubt we will ever have more than 16 million object simultaneous  instances, if we do we can always change this :P but giving priority (more bits) to the validator is usually better.